package.json

```diff
@@ -15,8 +15,9 @@
     "lint": "eslint .",
     "test": "nyc ava test/index.test.js",
     "remove-dist": "rimraf dist/*",
+    "copy-types": "ncp src/index.d.ts dist/index.d.ts",
```

Comment:
This isn't needed.

We can keep this at the root and add the following in package.json

```diff
+ "typings": "./index.d.ts"
```

Reply:
Thanks for your reminding about the `typings` field, I totally forgot that we don't need to ship source code and type declaration side-by-side.